Resources/Content/MSGraph/Applications/Simeon Cloud Sync.json

```diff
@@ -58,18 +58,6 @@
           "id": "e4aa47b9-9a69-4109-82ed-36ec70d85ff1",
           "type": "Scope"
         },
-        {
-          "id": "7b8a2d34-6b3f-4542-a343-54651608ad81",

```

Comment:
Would you be willing to help me understand why we are removing this one?

Reply:
We should not....checking why it got removed